index.bs

```diff
@@ -14896,12 +14896,12 @@ text is “<span class="input">a1</span>”, it is tokenized as a single <emu-t
 and not as a separate <emu-t class="regex"><a href="#prod-identifier">identifier</a></emu-t> and <emu-t class="regex"><a href="#prod-integer">integer</a></emu-t>.
 If the longest possible match could match one of the above named terminal symbols or
 one of the other terminal symbols from the grammar, it must be tokenized as the latter.
-Thus, the input text “<span class="input">long</span>” is tokenized as the quoted terminal symbol
+Thus, the input text “<span class="input">long</span>” is tokenized as the terminal symbol
 <emu-t>long</emu-t> rather than an <emu-t class="regex"><a href="#prod-identifier">identifier</a></emu-t> called "<code>long</code>",
-and “<span class="input">.</span>” is tokenized as the quoted terminal symbol
+and “<span class="input">.</span>” is tokenized as the terminal symbol
```

Comment:
The example in this line is not exactly a “real” terminal symbol of the grammar. Though it’s still present as an alternative of `Other`, it’s not used elsewhere and therefore has no meaning — if it were removed, everything would stay exactly the same because  `Other` would still include it via `other`, which is also only used in `Other` and only currently _doesn’t_ include `.` it  because `Other` _does_ include it. :)

Though the example was already present, the new language seems to make this situation even more confusing and probably justifies removing the vestigial `.` and `-` alternatives of `Other`.

Reply:
Wanna create a follow-up PR @bathos? :-)

Reply:
Afraid I’m [unable to](https://github.com/whatwg/sg/issues/93#issuecomment-1120799691).